lib/quickbooks-ruby.rb

```diff
@@ -226,6 +226,7 @@ class ThrottleExceeded < Forbidden; end
   class TooManyRequests < StandardError; end
   class ServiceUnavailable < StandardError; end
   class MissingRealmError < StandardError; end
+  class UnhandledHttpRedirect < StandardError; end
```

Comment:
Not specific to this PR but I'm wondering what @ruckus would think about introducing a base QBO error that all of these can inherit from so there's an single one to target for a rescue block, e.g:
```
class Error < StandardError; end
class InvalidModelException < Error; end
class AuthorizationFailure < Error; end
...
```
Then you can just catch all of these with:
```
rescue Quickbooks::Error => e
```
```

Reply:
@drewish thats probably a good idea. A PR would be much appreciated. Thank you!

Reply:
@ruckus Great, I opened up https://github.com/ruckus/quickbooks-ruby/pull/445

Reply:
That was merged so this PR will need a rebase now.